### PR TITLE
[TFLite] Support for broadcasting for Add,Mul,Div,Sub,Min,Max.

### PR DESCRIPTION
**Pull request by vuzelac-cadence**
TFLite is missing proper broadcasting for binary operators.

Here, adding a few operators for which OperatorTests exist, that use explicit output type as that's what's needed for quantized types.

Later, the support can be extended to other operators after OperatorTests are added.

### CODE REVIEW

File: utils/scripts/gen_tflite_models.py [hunk 1]
```diff
@@ -373,6 +373,67 @@ def gen_binary_operator_test(name, type, input_shape):
...
+    model = Model(inputs=[inp1, inp2], outputs=[out])
+    # Create data.
+    np.random.seed(0)
+    inp1_tensor = np.random.rand(*shape_1).astype(np.float32)
+    inp2_tensor = np.random.rand(*shape_2).astype(np.float32)
+    if type == "pow":
+        inp1_tensor = np.abs(inp1_tensor) + 1
+    out_tensor = model.predict([inp1_tensor, inp2_tensor])
+    # Save model.
+    dataset_gen = DatasetGenerator([inp1_tensor, inp2_tensor])
```
Comment (hl475) [inline utils/scripts/gen_tflite_models.py:407]: Is `DatasetGenerator` defined somewhere? Are you using `dataset_gen` later in the code?

Comment (vuzelac-cadence) [reply utils/scripts/gen_tflite_models.py]: Removed

**Review by hl475 — APPROVED** [review]
LGTM, thanks!

**Comment by mciprian13** [comment]
@vuzelac-cadence Can you provide some more details on why the previous broadcasting method was not correct? Also why your broadcasting method is not applicable to all the binary operators? At this point this selective change seems hackish to me ...

**Comment by vuzelac-cadence** [comment]
@mciprian13 current code was broadcasting the rank only, not doing the proper broadcasting per Tensorflow spec. 
So, the previous method was just not complete, IMO.

As for applying to all operators. I didn't quite get what is hackish here. I applied it to an operator I need (Mul) and few others that came for free (not much changes needed). 
For others, there are more changes needed in Glow which I was not ready to do.

If you want to say that you wouldn't accept a change that doesn't covers all the possible operators that's fine. I don't agree with that but still it's a valid choice when approving an MR. 

But also, note that many things are introduced without the full support, like Importers themselves. I do understand this is not a fair comparison but if we go to operators one by one, many of them are missing this or that.  None of that looks hackish to me. Operators features are added one by one, when needed. Looks pretty standard to me...

**Pull request closed without merging**